bedrock/firefox/templates/firefox/home/index-master.html

```diff
@@ -0,0 +1,292 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% add_lang_files "firefox/home-master" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+{% from "macros.html" import fxa_cta_link with context %}
+{% from "macros-protocol.html" import call_out, call_out_compact, feature_card with context %}
+
+{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+{% if LANG == 'en-CA' %}
+  {% set title_suffix = 'Firefox (CA)' %}
+{% elif LANG == 'en-GB' %}
+  {% set title_suffix = 'Firefox (UK)' %}
+{% else %}
+  {% set title_suffix = 'Firefox' %}
+{% endif %}
+{% block page_title %}{{ _('Protect your life online with Firefox and a family of privacy-first products') }}{% endblock %}
```

Comment:
They've changed this title to "Firefox - Protect your life online with privacy-first products" (shorter for SEO)